packages/webapp/src/pages/Integrations/providerConfigKey/Show.tsx

```diff
@@ -59,16 +63,20 @@ export const ShowIntegration: React.FC = () => {
     const onEvent: OnConnectEvent = useCallback(
         (event) => {
             if (event.type === 'close') {
+                invalidateConnections(cache, mutate);
```

Comment:
should not invalidate here, just on connect should be enough because you can close without connecting

Reply:
Fixed in 48b412cc